Controllers/articlesController.js

```diff
@@ -1,8 +1,9 @@
 const {selectArticles} = require('../Models/index')
 const {selectArticlesById} = require('../Models/index')
 const {selectCommentsForArticle} = require('../Models/index');
-const {checkArticleExists} = require('../Models/selectArticles');
-const {insertComment} = require('../Models/index')
+const {checkArticleExists} = require('../Models/index');
+const {insertComment} = require('../Models/index');
+const {updateArticle} = require('../Models/index')
```

Comment:
You can require these all on the same line but putting them in the same object and comma separating them, since they come from the same file :)